applications/luci-app-frpc/htdocs/luci-static/resources/view/frpc.js

```diff
@@ -135,7 +135,7 @@ function getServiceStatus() {
 
 function renderStatus(isRunning) {
 	var renderHTML = "";
-	var spanTemp = "<span style=\"color:%s;font-weight:bold;margin-left:15px\">%s - %s</span>";
+	var spanTemp = "<em><b><font color=%s>%s %s</font></b></em>";
```

Comment:
Hm, `<font>` tags are long deprecated, could you replace this with `<span style="color:%s">%s %s</span>` please? Also use `<strong>` instead of `<b>`.

Reply:
I see. It must be like this, right?
```
"<em><span style=\"color:%s;\"><strong>%s %s</strong></span></em>"
```

Reply:
You can also enclose the string in single quotes, this way you do not need to escape the double quotes. Also the final semicolon in the `style="..."` attribute can be omitted:

```javascript
'<em><span style="color:%s"><strong>%s %s</strong></span></em>'
```

Reply:
oh my fault. Done it.